020-Getting-started/070-nextjs.mdx

```diff
@@ -164,43 +190,30 @@ export default function PostsListPage() {
 }
 ```
 
-This error is because Xata stores date fields as an object. To use it within this application we'll need to serialize it to a string before it can be used.
-
-```tsx {6}
-// ...
-records: records.map((record) => ({
-  ...record,
-  // We need to convert our date from an object to a string to use it here
-  // If you didn't set up your pubDate column as "not null" Typescript may complain, so we'll add ? after pubDate for this example
-  pubDate: record.pubDate?.toDateString(),
-})),
-// ...
-```
-
 On save our page should contain no errors. The last thing to do is pass our newly created server-side props to the render so we can see the results on the page.
 
-```tsx {16-32}
-import { getXataClient, Posts } from '../xata';
+```tsx {17-36}
+import { getXataClient, PostsRecord } from '../xata';
+// We'll assign the client to xata to make it easier to call
 const xata = getXataClient();
 
+// We will make the query to Xata on the server.
+// This will allow the query to read the API_KEY from the .env file
 export const getServerSideProps = async () => {
-  const records: Posts[] = await xata.db.Posts.getMany();
+  const records = await xata.db.Posts.getMany();
```

Comment:
While we're here, does it make sense to change to `getAll` or `getPaginated` instead of `getMany`?

Reply:
I'm unsure about opening that discussion now, ha ha, I'm kind of in the `getMany` team. I'd rather merge this PR as it is, and have that discussion separately if we want.